chokuretsu/guide/play-game.html

```diff
@@ -0,0 +1,241 @@
+                        <li>Wait for the firmware dump to complete. Then, power off your Nintendo DS, remove the cart
+                            and insert the flash cart's SD card into your computer (or connect it with the correct
+                            cable).
+                        </li>
+                        <li>Your flash cartridge's SD card/storage will now contain <code>BIOSNDS7.ROM</code>, <code>BIOSNDS9.ROM</code>
+                            and a <code>.BIN</code> file starting with the letters "FW". These are your Nintendo DS'
+                            ARM7 and ARM9 BIOS files and its firmware respectively.
+                        </li>
+                    </ol>
+                </li>
+                <li>Copy your ARM7, ARM9 and firmware files into the same directory as where you downloaded melonDS
+                    to.
+                </li>
+                <li>Open melonDS, then on the top bar menu click "Config" <i class="fa-solid fa-arrow-right"></i> "Emu
+                    settings"
+                </li>
+                <li>On the "General" tab, ensure Console Mode is set to "DS". Click through to the tab named "DS-mode"
+                </li>
+                <li>On each of the required BIOS and firmware files, select the ARM7 <code>.ROM</code>, ARM9
+                    <code>.ROM</code> and firmware <code>.BIN</code> files you ripped from your system.
+                </li>
+            </ol>
+            <b>Playing the game:</b>
+            <p>With melonDS setup, all you need to do in order to play the game is select "File" <i
+                    class="fa-solid fa-arrow-right"></i> "Open ROM" in the top menu bar and choose the ROM file you
+                patched earlier.</p>
+            <p>melonDS has a number of advanced features, such as configuring controllers and tweaking display
+                settings. You can check out the <a href="https://melonds.kuribo64.net/" target="_blank"
+                                                   class="exclude-device-href">melonDS Website</a> for more information.
+            </p>
+
+            <h4>Other PC emulators</h4>
+            <ul>
+                <li><a href="https://desmume.org/" target="_blank" class="exclude-device-href">DeSumME</a>&#65293;Widely
+                    popular with good community support/controller compatibility.
+                </li>
+            </ul>
+
+            <!-- Wii U VC emulator -->
+            <h3 id="wii-u-vc">Emulate on Wii U by injecting the Virtual Console</h3>
+            <p>If you own a Wii U console, Nintendo's official Virtual Console DS emulator provides accurate emulation,
+                complemented nicely by the Wii U Gamepad, which works great with the touchscreen used frequently in
+                this game.</p>
+
+            <!-- Emulators on mobile devices -->
+            <h3 id="mobile-emulators">Emulate on mobile devices</h3>
+            <p>TBD :-)</p>
```

Comment:
Looked into this. There's basically two options that are worth talking about:

* DraStic
* RetroArch's (DeSMuME core)

RetroArch is basically the only worthwhile option for iOS while DraStic is apparently superior for Android but costs $5. I haven't bought it but I'd be willing to do so and write up some instructions for people here.